.engram/manifest.md

```
@@ -0,0 +1,330 @@
+| Tokenizer Manager | 1 | 9 |
+| HTTP Server | 1 | 3 |
+| Server Args | 1 | 3 |
+| Mamba Radix Cache | 1 | 3 |
+| IO Struct | 1 | 3 |
+| Model Runner KV Cache Mixin | 1 | 2 |
+| Scheduler Output Processor | 1 | 3 |
+| Schedule Batch | 1 | 2 |
+| Model Config | 1 | 2 |
+
+### Other Modified (fewer blocks): 24 total
+| File | Blocks |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)